curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-localstorage-by-building-a-todo-app/64e4e78a7ea4a168de4e6a38.md

```diff
@@ -27,10 +37,10 @@ Your event listener should listen for a `click` event.
 assert.match(code, /openTaskFormBtn\.addEventListener\(\s*('|"|`)click\1/)
 ```
 
-Your event listener should include a callback function for the second argument using arrow syntax without curly braces. Then you should use the `classList` property and its `toggle` method on the `taskForm`, to toggle the class `hidden`.
+Your event listener's callback function should contain `taskForm.classList.toggle("hidden")`. 
 
 ```js
-assert.match(code, /openTaskFormBtn\.addEventListener\(\s*('|"|`)click\1\s*,\s*\(\)\s*\s*=>\s*taskForm\.classList\.toggle\(\s*('|"|`)hidden\2\s*\)\s*\);?/)
+assert.match(code, /openTaskFormBtn\.addEventListener\(\s*('|"|`)click\1\s*,\s*\(\)\s*=>\s*taskForm\s*\.classList\s*\.\s*toggle\s*\(\s*('|"|`)hidden\2\s*\)\s*;?\s*\);?/)
```

Comment:
@gikf  what do you think of this solution? i simplified it a little bit.
```suggestion
assert.match(code, /openTaskFormBtn\.addEventListener\(\s*['"`]click['"`]\s*,\s*\(\s*\)\s*=>\s*{?\s*taskForm\.classList\.toggle\(\s*['"`]hidden['"`]\s*\)\s*;?\s*}?\s*\)\s*;?/)
```

Reply:
My problem with it is allowing mixed up quotes and passing with `{` or `}` just on one side. For example, this would match:

```javascript
openTaskFormBtn.addEventListener(`click", () => {
  taskForm.classList.toggle(`hidden')
);
```

Reply:
@gikf what would be your simplified solution in this case ? also can you drop your twitter handle ? whould like to say hello

Reply:
Well, I don't know how to make it simpler, while keeping the correctness, so I wouldn't :slightly_smiling_face: 

I'm not using twitter at this time, you can find me on discord - sanity (or not)